tests/unit/dataactvalidator/test_a33_appropriations_1.py

```diff
@@ -18,38 +18,70 @@ def test_column_headers(database):
     assert (actual & expected_subset) == expected_subset
 
 
-def test_success(database):
+def test_success_populated_ata(database):
     """ Tests that TAS for SF-133 are present in File A """
     submission_id = randint(1000, 10000)
     tas, period, year, code = 'some-tas', 2, 2002, 'some-code'
+
     sf1 = SF133Factory(tas=tas, period=period, fiscal_year=year,
-                       agency_identifier=code)
+                       allocation_transfer_agency=code, agency_identifier=code)
     sf2 = SF133Factory(tas=tas, period=period, fiscal_year=year,
-                       agency_identifier=code)
-    submission = SubmissionFactory(
-        submission_id=submission_id, reporting_fiscal_period=period,
-        reporting_fiscal_year=year, cgac_code=code)
+                       allocation_transfer_agency=code, agency_identifier=code)
```

Comment:
The null ATA case is tested below, so maybe just change the AID here to not match so we're sure it's using the ATA